clients/python/sysadminctl

```diff
@@ -112,6 +119,8 @@ class SetCommand(object):
         self.parser.add_argument("--type", type=str, default="str",
                                  choices=type_choices,
                                  help="value type, default: str")
+        self.parser.add_argument("-v", "--verbose", action='store_true',
+                                 help="get output with variable type")
```

Comment:
IMO, you shouldn't be this specific with what this does. I would leave it as "Enable verbose output" in case we add some other stuff later. Same for modify.